bot/cogs/free.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from datetime import datetime
+
+from discord import Colour, Embed, Member, utils
+from discord.ext.commands import BucketType, Context, command, cooldown
+
+
+log = logging.getLogger(__name__)
+
+
+class Free:
+    """Tries to figure out which help channels are free."""
+
+    PYTHON_HELP_ID = 356013061213126657
```

Comment:
It may be a good idea to move these settings to config-default.yml where similar settings are stored (like the channel ids and timeout/interval settings). That keeps all the configuration stuff/constants in one place.

Reply:
I've moved the ID to the config, but I'm wondering if moving the timeout constant there is overkill? This command ultimately is temporary, and I can't imagine this timeout will be used for anything else.

Reply:
Centralizing constants means we don't need to hunt through cogs to find their configuration options if we want to change them.

Reply:
Fair enough, I'll add the timeout to the config as well.